### DISCUSSION TITLE
Mentioning/acknowledging OHBM BrainHack on our webpage / GitHub readme

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#139] [Ideas]
As the BrainHack had a major role in pushing the project forward (that is how Steffen and I met), I think we should acknowledge BrainHack (2020 and 2021 editions)

**Comment by aswinnarayanan** [comment]
That sounds good @civier. Best place to add is would be the landing site [here](https://github.com/NeuroDesk/neurodesk.github.io)
Please feel free to add it
